src/components/VoteCell.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import styled from 'styled-components';
+import axios from 'axios';
+
+export default function VoteCell(props) {
```

Comment:
VotePage에서 넘겨준걸 이렇게 비구조화할당해서 받으면 되겠죠??

```suggestion
export default function VoteCell({rank,person}) {
```